doc/appendices/command-line/traffic_ctl.en.rst

```diff
@@ -258,27 +258,54 @@ traffic_ctl host
     Get the current status of the hosts used in parent.config as a next hop in a multi-tiered cache hierarchy.  The value 0 or 1 is returned indicating that the host is marked as down '0' or marked as up '1'.  If a host is marked as down, it will not be used as the next hop parent, another host marked as up will be chosen.
 
 .. program:: traffic_ctl host
-.. option:: down --time seconds --reason 'manual|active|local' HOSTNAME [HOSTNAME ...]
+.. option:: down --time seconds --reason 'all|active|local|manual|self_detect' HOSTNAME [HOSTNAME ...]
```

Comment:
What is the reason for having an option "all" while marking a host down?

Reply:
what is the reason to have self_detect as an option? shouldn't self_detect be identified by ATS automatically?

Reply:
I added the self_detect reason.  When parent selection reads the parent.config and it identifies that a parent is itself, it marks it down if proxy.config.http.parent_proxy.self_detect is set to 2.  previously it was marked down due to the manual reason.  I added in the self_detect in an earlier pr because I believe we need to distinguish that it was marked down due to self detection.